code/__HELPERS/unsorted.dm

```diff
@@ -923,13 +923,19 @@ var/list/WALLITEMS_INVERSE = typecacheof(list(
 
 //Get the dir to the RIGHT of dir if they were on a clock
 //NORTH --> NORTHEAST
-/proc/get_clockwise_dir(dir)
-	. = angle2dir(dir2angle(dir)+45)
+/proc/get_clockwise_dir(dir, cardinal = FALSE)
```

Comment:
firstly, no.
secondly use ```turn()``` where you need to.

I only made these clockwise procs as shortcuts to +45, they should also really be macros so now that's your job
```#define GET_CLOCKWISE_DIR(dir) turn(dir, 45)```, please update their calls accordingly to.